src/rnanorm/methods/between_sample.py

```diff
@@ -72,7 +73,9 @@ def _get_norm_factors(self, X: Numeric2D) -> Numeric1D:
         :param X: Expression raw count matrix (n_samples, n_features)
         """
         X = remove_zero_genes(X)
-        lib_size = LibrarySize().fit_transform(X)
+        # Make sure that global set_config(transform_output="pandas")
```

Comment:
Is this really necessary? It's kind off ugly to break the intended sklearn logic.